streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -516,6 +517,122 @@ public synchronized Topology addSink(final String name,
         return this;
     }
 
+    /**
+     * Add a new sink that forwards records from upstream parent processor and/or source nodes to Kafka topics dynamically.
```

Comment:
IMHO, "dynamically" need a little bit more explanation. Should we also state, that all those topics must be created by the user manually?